layouts/index.hbs

```diff
@@ -140,18 +148,18 @@
         }
       }
     </style>
-    
+
     <div class="interactive">
       <span id="interactive-first">
         <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
       </span>
-      <a href="http://events.linuxfoundation.org/events/node-interactive-europe">
+      <a id="interactive-left" href="http://events.linuxfoundation.org/events/node-interactive-europe">
```

Comment:
Hey @lpinca I ended up merging this as is due to the release tonight. Do you think it would make sense to submit that optimization in another pr?